### PR TITLE
Port Android Sample SwitchDemo

### PR DESCRIPTION
**Pull request by gal42**

### CODE REVIEW

File: SwitchDemo/SwitchDemo/Activity1.cs
Comment (rachelkang) [inline SwitchDemo/SwitchDemo/Activity1.cs:16]: nitpick: Remove extra blank lines

File: SwitchDemo/SwitchDemo/Activity1.cs
Comment (rachelkang) [inline SwitchDemo/SwitchDemo/Activity1.cs:33]: nitpick: Remove extra blank line

File: SwitchDemo/SwitchDemo/Activity1.cs
Comment (rachelkang) [inline SwitchDemo/SwitchDemo/Activity1.cs:35]: nitpick: Remove extra blank line

**Review by rachelkang — CHANGES_REQUESTED** [review]
Great work so far! Left some comments on how you can improve further :)

File: SwitchDemo/README.md
Comment (rachelkang) [inline SwitchDemo/README.md:3]:
```suggestion
description: "Shows how to use a switch control"
```

File: SwitchDemo/README.md
Comment (rachelkang) [inline SwitchDemo/README.md:11]: again, ice cream sandwich is old news! But let's add `ui` since the metadata shows a tag for "User Interface"

```suggestion
    - ui
```

File: SwitchDemo/README.md
Comment (rachelkang) [inline SwitchDemo/README.md:22]: Thanks for updating the screenshots!!! :)

**Review by rachelkang — CHANGES_REQUESTED** [review]
Just a few more nitpicky comments!

File: SwitchDemo/SwitchDemo/Activity1.cs
Comment (gal42) [reply SwitchDemo/SwitchDemo/Activity1.cs]: Hi Rachel, I updated the sample with the changes

Comment (rachelkang) [reply SwitchDemo/SwitchDemo/Activity1.cs]: Awesome! Thanks for the speedy update :)

**Review by rachelkang — APPROVED** [review]
LGTM! Builds and deploys as expected. 
Let's get one more review from the Android team before merging!

File: SwitchDemo/SwitchDemo/Activity1.cs
Comment (jonathanpeppers) [inline SwitchDemo/SwitchDemo/Activity1.cs:15]: Can you use the newer C# style, like:
```suggestion
            s.CheckedChange += (sender, e) => {
```

File: SwitchDemo/SwitchDemo/Activity1.cs
Comment (jonathanpeppers) [inline SwitchDemo/SwitchDemo/Activity1.cs:19]: This one is weird... This doesn't look like it can return null, but Google didn't decorate it with `@NotNull`:

https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/widget/Toast.java;l=492

What you did here is great! I'm just thinking if there is something we should do to make this better in xamarin/xamarin-android.

File: SwitchDemo/SwitchDemo/Activity1.cs
Comment (gal42) [reply SwitchDemo/SwitchDemo/Activity1.cs]: Hi Jonathan! I just updated the file

**Review by jonathanpeppers — APPROVED** [review]

**Pull request merged**
